src/frontend/display/btmodelviewreaddisplay.h

```diff
@@ -207,4 +211,5 @@ class BtModelViewReadDisplay : public QWidget {
     BtQmlScrollView* m_widget;
     QString m_currentAnchorCache;
 
+    std::unique_ptr<QTextToSpeech> m_textToSpeech;
```

Comment:
I think it might make more sense to create a single `QTextToSpeech` instance for the whole application, instead one for each `BtModelViewReadDisplay`. I suggest moving this to the main window class (`BibleTime`).

Reply:
Done.